Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM dunglas/frankenphp
```

Comment:
A image version should be defined. Otherwise mismatching PHP versions could occur.